yarn.lock

```diff
@@ -0,0 +1,2427 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
a yarn.lock has to be used to ensure the tests run with the correct version of babel-helper-fixtures for the submodule.